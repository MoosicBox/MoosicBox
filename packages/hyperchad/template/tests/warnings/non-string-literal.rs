use hyperchad_template::container;

fn main() {
    container! {
        42
        42usize
        42.0
        'a'
        b"a"
        b'a'

        // `true` and `false` are only considered literals in attribute values
        input type=text disabled=true;
        input type=text disabled=false;
    };
}
