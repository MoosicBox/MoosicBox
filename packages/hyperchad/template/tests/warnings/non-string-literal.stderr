error: literal must be double-quoted: `"a"`
 --> tests/warnings/non-string-literal.rs:8:9
  |
8 |         'a'
  |         ^^^

error: expected string, integer, or float
 --> tests/warnings/non-string-literal.rs:9:9
  |
9 |         b"a"
  |         ^^^^

error: expected string, integer, or float
  --> tests/warnings/non-string-literal.rs:10:9
   |
10 |         b'a'
   |         ^^^^
